### PR TITLE
Secure cookies

### PR DESCRIPTION
**Pull request by camallen**

### CODE REVIEW

File: config/initializers/session_store.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
-
-Rails.application.config.session_store :cookie_store, key: '_Panoptes_session'
+secure_option = (Rails.env.development? || Rails.env.test?) ? false : true
```
Comment (astopy) [inline config/initializers/session_store.rb]: Maybe `secure_option = not (Rails.env.development? || Rails.env.test?)`

**Comment by marten** [comment]
wouldnt setting config.force_ssl in the environments have the same effect?

**Comment by camallen** [comment]
yeah that setting would but don't we want to terminate ssl layer at the CDN?

**Comment by camallen** [comment]
going to use the force_ssl config as it provides a fall back mechanism should the site be deployed without an ssl termination in front of it.

**Comment by marten** [comment]
Cloudfront will set the correct header which Rails looks for: http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/x-forwarded-headers.html#x-forwarded-proto

**Comment by camallen** [comment]
Thanks - just found the relevant rack source, https://github.com/josh/rack-ssl/blob/master/lib/rack/ssl.rb#L41

**Comment by marten** [comment]
LGTM

**Pull request merged**